<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>studenta</class>
 <widget class="QWidget" name="studenta">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1076</width>
    <height>651</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>130</y>
     <width>651</width>
     <height>441</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QLineEdit" name="leFio">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>140</y>
      <width>181</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lePhone">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>190</y>
      <width>181</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="leBd">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>240</y>
      <width>181</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="leInfo">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>300</y>
      <width>181</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QTableWidget" name="twStudent">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>20</y>
      <width>401</width>
      <height>311</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextEdit" name="teStudent">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>380</y>
      <width>591</width>
      <height>51</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="lbInfoStA">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>280</y>
      <width>101</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Информация</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbBdStA">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>220</y>
      <width>121</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Дата рождения</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbPhoneStA">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>170</y>
      <width>121</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Номер телефона</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbFIOStA">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>120</y>
      <width>67</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>ФИО</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnAddStA">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>340</y>
      <width>89</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Добавить</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnDelStA">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>340</y>
      <width>89</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Удалить</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnSelectAllStA">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>340</y>
      <width>89</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Выбрать все</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnBackStA">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>340</y>
      <width>89</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Назад</string>
    </property>
   </widget>
   <widget class="QPushButton" name="bntConnect">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>340</y>
      <width>101</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Подключение</string>
    </property>
   </widget>
   <widget class="QPushButton" name="bntEdit">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>340</y>
      <width>89</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Изменить</string>
    </property>
   </widget>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>90</y>
     <width>111</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Ученики</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
